make manual fails on "sigmap_test.v"

## Steps to reproduce the issue

```
git clone https://github.com/yosyshq/yosys
git checkout 61324cf55 # master branch tip at the time of this writing
make all
# I don't have luximono on Fedora, replace with libertine instead:
for f in `find manual -name \*.tex -exec grep -l {luximono} {} \;`; do
    sed -i 's|{luximono}|{libertine}|' $f
done
make manual
```

## Expected behavior

PDF manual build completes successfully

## Actual behavior

manual build errors out with the following log output:

```
[...]
make[1]: Entering directory '/home/somlo/yosys/manual/PRESENTATION_Prog'
../../yosys-config --exec --cxx -Wall -Wextra -ggdb -I../../share/include -MD -MP -D_YOSYS_ -fPIC -I/usr/local/include -std=c++11 -Os -DYOSYS_ENABLE_READLINE -DYOSYS_ENABLE_PLUGINS -DYOSYS_ENABLE_GLOB -DYOSYS_ENABLE_ZLIB -DYOSYS_ENABLE_TCL -DYOSYS_ENABLE_ABC -DYOSYS_ENABLE_COVER --ldflags -o my_cmd.so -shared my_cmd.cc --ldlibs
my_cmd.cc:18:21: warning: format specifies type 'ssize_t' (aka 'long') but the argument has type 'int' [-Wformat]
                    GetSize(mod->wires()), GetSize(mod->cells()));
                    ^~~~~~~~~~~~~~~~~~~~~
my_cmd.cc:18:44: warning: format specifies type 'ssize_t' (aka 'long') but the argument has type 'int' [-Wformat]
                    GetSize(mod->wires()), GetSize(mod->cells()));
                                           ^~~~~~~~~~~~~~~~~~~~~
2 warnings generated.
../../yosys -Ql test0.log_new -m ./my_cmd.so -p 'my_cmd foo bar' absval_ref.v

-- Parsing `absval_ref.v' using frontend ` -vlog2k' --

1. Executing Verilog-2005 frontend: absval_ref.v
Parsing Verilog input from `absval_ref.v' to AST representation.
Storing AST representation for module `$abstract\absval_ref'.
Successfully finished Verilog frontend.

-- Running command `my_cmd foo bar' --
Arguments to my_cmd:
  my_cmd
  foo
  bar
Modules in current design:
  $abstract\absval_ref (0 wires, 0 cells)

End of script. Logfile hash: a25069ff9d, CPU: user 0.01s system 0.00s, MEM: 8.97 MB peak
Yosys 0.13+3 (git sha1 61324cf55, clang 12.0.1 -fPIC -Os)
Time spent: 83% 2x read_verilog (0 sec), 14% 1x my_cmd (0 sec), ...
mv test0.log_new test0.log
../../yosys -Ql test1.log_new -m ./my_cmd.so -p 'clean; test1; dump' absval_ref.v

-- Parsing `absval_ref.v' using frontend ` -vlog2k' --

1. Executing Verilog-2005 frontend: absval_ref.v
Parsing Verilog input from `absval_ref.v' to AST representation.
Storing AST representation for module `$abstract\absval_ref'.
Successfully finished Verilog frontend.

-- Running command `clean; test1; dump' --
Name of this module: absval

autoidx 4

module \absval

  wire width 4 $auto$my_cmd.cc:41:execute$1

  wire width 4 output 2 \y

  wire width 4 input 1 \a

  cell $mux $auto$my_cmd.cc:43:execute$3
    parameter \WIDTH 4
    connect \Y \y
    connect \S \a [3]
    connect \B $auto$my_cmd.cc:41:execute$1
    connect \A \a
  end

  cell $neg $auto$my_cmd.cc:42:execute$2
    parameter \Y_WIDTH 4
    parameter \A_WIDTH 4
    parameter \A_SIGNED 1
    connect \Y $auto$my_cmd.cc:41:execute$1
    connect \A \a
  end
end

attribute \cells_not_processed 1
attribute \src "absval_ref.v:1.1-3.10"
module $abstract\absval_ref
end

End of script. Logfile hash: 01cda1039c, CPU: user 0.00s system 0.00s, MEM: 8.88 MB peak
Yosys 0.13+3 (git sha1 61324cf55, clang 12.0.1 -fPIC -Os)
Time spent: 63% 1x clean (0 sec), 20% 2x read_verilog (0 sec), ...
mv test1.log_new test1.log
../../yosys -Ql test2.log_new -m ./my_cmd.so -p 'test2' sigmap_test.v

-- Parsing `sigmap_test.v' using frontend ` -vlog2k' --

1. Executing Verilog-2005 frontend: sigmap_test.v
Parsing Verilog input from `sigmap_test.v' to AST representation.
Storing AST representation for module `$abstract\test'.
Successfully finished Verilog frontend.

-- Running command `test2' --
terminate called after throwing an instance of 'std::out_of_range'
  what():  dict::at()
make[1]: *** [Makefile:19: test2.log] Aborted (core dumped)
make[1]: Leaving directory '/home/somlo/yosys/manual/PRESENTATION_Prog'
make: *** [Makefile:897: manual] Error 2
```

--- a/manual/PRESENTATION_Prog/Makefile
+++ b/manual/PRESENTATION_Prog/Makefile
@@ -16,6 +16,6 @@ test1.log: my_cmd.so
 	mv test1.log_new test1.log
 
 test2.log: my_cmd.so
-	../../yosys -Ql test2.log_new -m ./my_cmd.so -p 'test2' sigmap_test.v
+	../../yosys -Ql test2.log_new -m ./my_cmd.so -p 'hierarchy -top test; test2' sigmap_test.v
 	mv test2.log_new test2.log
 

